packages/instantsearch.js/src/connectors/breadcrumb/connectBreadcrumb.ts

```diff
@@ -194,9 +195,14 @@ const connectBreadcrumb: BreadcrumbConnector = function connectBreadcrumb(
             return [];
           }
 
-          const [{ name: facetName }] = state.hierarchicalFacets;
+          const facet = find(state.hierarchicalFacets, (f) =>
+            Boolean(f.separator)
```

Comment:
here you're checking if it has a separator (I guess because menu doesn't have one). Could we check if the attributes match maybe?

Reply:
Yeah, this was more a poc change to see if the resulting functionality of the widgets is correct. You can check via the sandbox linked above. Basically breadcrumb will only pick up hierarchical menu facets and ignore menu ones.

I think a better way would be to use `hierarchicalFacetName` which is used throughout the connector. I've made this change in the latest commit, let me know if you think it's correct. I'll add some test cases for it in that case.

Reply:
Seems to have been this way since `getWidgetRenderState` was implemented for breadcrumbs.
Could it be the case that breadcrumb needs to also display menu attributes as well? Otherwise seems like an oversight and we caught it now since using both hierarchical menu and a normal menu isn't that common I believe

Reply:
no, breadcrumb is only meant to work with hierarchical menu. I guess it was an oversight since until recently you had to mount them in the right order and breadcrumb didn't work standalone either. Let's go!